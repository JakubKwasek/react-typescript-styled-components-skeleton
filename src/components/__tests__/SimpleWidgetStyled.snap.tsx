import * as React from "react";
import * as renderer from "react-test-renderer";
// import { SimpleWidgetStyled } from "../styled";
// import "jest-styled-components";

// describe("SimpleWidgetStyled", () => {
// 	it("renders correctly", () => {
// 		const tree = renderer
// 			.create(<SimpleWidgetStyled />)
// 			.toJSON();
// 		expect(tree).toMatchSnapshot();
// 	});
// });
