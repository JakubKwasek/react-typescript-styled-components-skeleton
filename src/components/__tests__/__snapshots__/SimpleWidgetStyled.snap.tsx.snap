// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleWidget renders correctly 1`] = `
<div
  className=""
/>
`;
